### PR TITLE
Implement sendTokenToIntercom method on iOS side

### LINKED ISSUE #52: Error when using sendTokenToIntercom
Whenever I try to use the sendTokenToIntercom command I get the following error on ios: 
`MissingPluginException(No implementation found for method sendTokenToIntercom on channel maido.io/intercom.`

Same result for both 2.0.7 and 2.0.5-v2 and Flutter version 1.9.1

### LINKED ISSUE #71: MissingPluginException(No implementation found for method sendTokenToIntercom on channel maido.io/intercom)
Hi team,
I am getting the above error when calling .sendTokenToIntercom with an FCM token.

package v: intercom_flutter: ^2.2.0

### LINKED ISSUE #84: Unhandled Exception: MissingPluginException(No implementation found for method sendTokenToIntercom on channel maido.io/intercom)
When i run app in ios

### PR DESCRIPTION
**Pull request by indigothm**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: intercom_flutter
 description: Flutter plugin for Intercom integration. Provides in-app messaging
   and help-center Intercom services
-version: 2.2.0+1
+version: 2.2.1+1
```
Comment (Zazo032) [inline pubspec.yaml]: This should be `2.2.1`

File: example/ios/Flutter/Flutter.podspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#
```
Comment (Zazo032) [inline example/ios/Flutter/Flutter.podspec]: Could you remove this file from the PR?

**Review by Zazo032 — CHANGES_REQUESTED** [review]
Hi @indigothm! Thanks for the PR. Just a few nits. Also, could you state your changes in `CHANGELOG.md` and add a small test that verifies the method is being called properly? Some examples [here](https://github.com/v3rm0n/intercom_flutter/blob/master/test/intercom_flutter_test.dart).

File: example/ios/Flutter/.last_build_id [hunk 3]
```diff
@@ -0,0 +1 @@
+0c9059c70654b99fdd937e7ed9a0a0ff
```
Comment (Zazo032) [inline example/ios/Flutter/.last_build_id]: Could you add this file to `gitignore` and remove it from the PR?

**Comment by indigothm** [comment]
Sure, will do that

**Comment by acoutts** [comment]
Hi! Any progress on finishing this to get it merged in?

**Comment by indigothm** [comment]
Hey, got distracted from that task a bit. Will try to push it tomorrow

**Comment by indigothm** [comment]
@acoutts Good to go. Added the test and removed the extra files

**Comment by Zazo032** [comment]
Thanks @indigothm! Changes LGTM, could you document your change on `CHANGELOG.md` too?

**Comment by indigothm** [comment]
@Zazo032 Done

**Review by Zazo032 — APPROVED** [review]

**Pull request merged**